src/core/server/saved_objects/migrations/core/document_migrator.test.ts

```diff
@@ -748,10 +748,7 @@ describe('DocumentMigrator', () => {
         migrator.migrate(_.cloneDeep(failedDoc));
         expect('Did not throw').toEqual('But it should have!');
       } catch (error) {
-        expect(error.message).toMatchInlineSnapshot(`
-          "Failed to transform document smelly. Transform: dog:1.2.3
-          Doc: {\\"id\\":\\"smelly\\",\\"type\\":\\"dog\\",\\"attributes\\":{},\\"migrationVersion\\":{}}"
-        `);
+        expect(error.message).toMatchInlineSnapshot(`"Dang diggity!"`);
```

Comment:
nit: let's use `toBe` assertion, as we don't use any benefits of `toMatchInlineSnapshot` here.